torch/fx/experimental/proxy_tensor.py

```diff
@@ -509,6 +511,26 @@ def set_original_aten_op(func):
 
 
 
+# This mode is **only** used for pre_dispatch tracing.
+# In particular, we need to make sure that autograd/autocast API's
+# that do not desugar into dispatcher operators stay in the graph.
+class ProxyTorchFunctionMode(TorchFunctionMode):
```

Comment:
This name seems pretty misleading, I definitely don't want to run this for non-preautograd tracing no?